client/python/ovmsclient/README.md

```diff
@@ -18,4 +20,13 @@ If you're looking for a quick way to try `ovmsclient` out or want to look up how
 There are also Dockerfiles available that prepare Docker image with `ovmsclient` installed and ready to use.
 Simply run `docker build` with the Dockerfile of your choice to get the minimal image:
 - [Ubuntu 20.04 based image](Dockerfile.ubuntu)
-- [UBI 8.4 based image](Dockerfile.redhat)
\ No newline at end of file
+- [UBI 8.4 based image](Dockerfile.redhat)
```

Comment:
I think we are using UBI 8.5 starting from that release.

Reply:
from client/python/ovmsclient/Dockerfile.redhat (develop branch):
ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.4
ARG FINAL_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:8.4